src/connector.ts

```diff
@@ -525,11 +528,6 @@ export default {
             nClient = undefined;
         }
 
-        // If an active debug session exists, terminate it
-        if (vscode.debug.activeDebugSession) {
-            vscode.debug.activeDebugSession.customRequest(REQUESTS.SEND_TERMINATED_EVENT);
-        }
-
```

Comment:
It's a bit unclear to me why the change of method for reaching clj-kondo makes it unnecessary to clean up the debug session connection?

Reply:
Ah, now I see that this is just wired up differently now. =)